src/requirements/requirement-frontend-utils.ts

```diff
@@ -484,52 +484,6 @@ export function computeFulfillmentCoursesAndStatistics(
   };
 }
 
-/**
- * Find related requirement ids to opt out of. This maintains the invariant that
- * there are no constraint violations when adding a course to the semester.
- * It should be deprecated after the new add modal is implemented.
- */
-export function getRelatedRequirementIdsForCourseOptOut(
```

Comment:
This function is now useless 🍅 

To get it back in the future (for whatever reason), `git revert aaefb283ef8c12c9643d47c9e76df04aba364bc6` should do it.